mmaction/apis/inference.py

```diff
@@ -111,10 +124,15 @@ def inference_recognizer(model, video_path, label_path, use_frames=False):
         data = scatter(data, [device])[0]
 
     # forward the model
-    with torch.no_grad():
-        scores = model(return_loss=False, **data)[0]
+    with OutputHook(model, outputs=outputs, as_tensor=True) as h:
```

Comment:
Seems that the default output is Tensor rather than numpy.dnarray as described in docstring.